package.nix

```diff
@@ -7,12 +7,6 @@ in
   nixpkgsPareto.overrideAttrs (oldAttrs: {
     src = ./.;
     version = "${builtins.hashFile "sha256" "${toString ./go.sum}"}";
-    subPackages = ["cmd/paretosecurity"];
-    nativeBuildInputs =
-      [
-        pkgs.pkg-config
-      ]
-      ++ oldAttrs.nativeBuildInputs;
```

Comment:
These are required in old gomodule (unrelated to unstable branch).

Reply:
Why aren't they required in nixpkgs?